string name
float32 range
---
geometry_msgs/Vector3[] relative_positions